astarte-device-DEVICE_ID_HERE-conf/config.json

```
@@ -0,0 +1,9 @@
+{
+  "realm": "REALM_HERE",
+  "device_id": "DEVICE_ID_HERE",
+  "pairing_url": "PAIRING_URL_HERE",
+  "credentials_secret": "CREDENTIALS_SECRET_HERE",
+  "astarte_ignore_ssl": true,
```

Use `false` as default value
```suggestion
  "astarte_ignore_ssl": false,
```